source/includes/_payment_methods.md

```diff
@@ -227,6 +227,88 @@ Attribute | Type | Description
 **method**| `string` | This must be `future_paypal`.
 
 
+## Stripe
+
+> Definition
+
+```
+POST https://api.shareactor.io/payment_methods
+```
+
+> Example request:
+
+``` http
+POST /payment_methods HTTP/1.1
+Content-Type: application/json
+Authorization: Bearer <jwt>
+X-Share-Api-Key: <shareactor-api-key>
+Host: api.shareactor.io
+
+{
+  "method": "stripe",
+  "token": "tok_1Bv5yfEeeXxFpLJtpWYeGYuy"
+}
+```
+
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "company": {
+    "$oid": "57ee9c71d76d431f8511142f"
+  },
+  "created": {
+    "$date": 1476118043580
+  },
+  "_id": {
+    "$oid": "<payment-method-id>"
+  },
+  "modified": {
+    "$date": 1476118043580
+  },
+  "deleted": false,
+  "user": {
+    "$oid": "57ee9c72d76d431f85111432"
+  },
+  "method": "stripe",
+  "name": "Stripe",
+  "customer_id": "cus_CJjTlT4Wci2P0u",
+  "token": "tok_1Bv5yfEeeXxFpLJtpWYeGYuy",
+  "card": {
+    'object': 'card',
+    'address_state': None,
+    'fingerprint': 'npPw68vQg1usKUWb',
+    'metadata': {},
+    'exp_year': 2019,
+    'country': 'US',
+    'last4': '4242',
+    'address_zip_check': None,
+    'address_zip': None,
+    'funding': 'credit',
+    'cvc_check': 'unchecked',
+    'id': 'card_1Bv5yfEeeXxFpLJtPBVfQ1wZ',
+    'tokenization_method': None,
+    'address_line1': None,
+    'exp_month': 12,
+    'brand': 'Visa',
+    'dynamic_last4': None,
+    'address_country': None,
+    'address_line2': None,
+    'address_line1_check': None,
+    'name': None,
+    'address_city': None
+  }
+}
+```
+
+This feature is only available for the mobile SDK.  To use it, the app sends the API an authorization code. The API is then exchanged for a Refresh Token, which is also stored in the object. When paying for an invoice, you should also add the `PayPal-Client-Metadata-Id` header to the API `/payments` in [Payments](#payments).
```

Comment:
this is not correct right? @GabrielBlondeau-Kirby

Reply:
Right, I did a mistake